src/rulesets/rest/2022-05-25/json-api-rules/__tests__/status-code-rules.test.ts

```diff
@@ -115,14 +115,19 @@ describe("status code rules", () => {
     const afterJson = {
       ...baseJson,
       paths: {
-        "/api/users/{user_id}": {
+        "/api/users": {
```

Comment:
This test case makes me realize we might need a rule added -- do not allow POSTs at all on a single resource (path ending with an id).